tests/filters/test_cpp.py

```diff
@@ -45,7 +45,7 @@ def test_cpp(self):
                 uint8_t tsdd = 3;
                 // {}
                 // {} \
-                reurn tsdd;
```

Comment:
Please undo the `reurn` changes, we are intentionally testing for this mispelling.

Reply:
OK!